.ci/check-style

```diff
@@ -37,3 +38,9 @@ if git --no-pager diff --diff-filter=d --name-only ${TARGET_BRANCH} HEAD | grep
 	git --no-pager diff
 	exit 1
 fi
+
+RUST_SOURCES=$(git --no-pager diff --diff-filter=d --name-only ${TARGET_BRANCH} HEAD | grep -E "^src/rust.*\.rs\$")
```

Comment:
Please exclude the vendor directory, unfortunately we can't control the deps' formatting :smile: